i18n/translations_test.go

```diff
@@ -0,0 +1,107 @@
+package i18n
+
+import (
+	"testing"
+
+	"github.com/nicksnyder/go-i18n/i18n/bundle"
+)
+
+var bobMap = map[string]interface{}{"Person": "Bob"}
+var bobStruct = struct{ Person string }{Person: "Bob"}
+
+var testCases = []struct {
+	id   string
+	arg  interface{}
+	want string
+}{}
+
+func init() {
+	initTestCases()
+}
+
+func initTestCases() {
+	b := bundle.New()
```

Comment:
It is used for creating a Tfunc and Tfunc is used for checking "d_days" in `init` (person_unread_email_count_timeframe).